src/main/scala/rocket/CSR.scala

```diff
@@ -594,7 +594,6 @@ class CSRFile(
     (if (fLen >= 32) "F" else "") +
     (if (fLen >= 64) "D" else "") +
     (if (usingVector) "V" else "") +
-    (if (usingBitManip) "B" else "") +
```

Comment:
Instead of deleting this, can you comment out a line, and add that the current spec does not define what sub-extensions constitute the 'B' misa bit?

Reply:
Done.